test/test_fx_experimental.py

```diff
@@ -157,5 +158,98 @@ def forward(self, a, b, offset):
         self.assertEqual(traced(a, b, offset), module_with_submodules(a, b, offset))
         assert len(module_with_submodules.graph.nodes) == 24
 
+    def test_assert_no_msg(self):
+
+        class M(torch.nn.Module):
+            def forward(self, a, b):
+                assert a == b
+                return a + b
+        m = M()
+        traced = symbolic_trace(m)
+
+        # Make sure the graph is well-formed
+        traced.graph.lint(traced)
+
+        # Check the IR to make sure there's a call_function node with target == "Assert"
+        self.assertTrue(any(node.op == 'call_function' and get_target_name(node.target) == "Assert" for node in traced.graph.nodes))
```

Comment:
Instead of using `get_target_name` here and below, can you just compare `node.target == torch.Assert`?